modules/tobira/pom.xml

```diff
@@ -22,6 +22,15 @@
       <artifactId>opencast-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.opencastproject</groupId>
+      <artifactId>opencast-workspace-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.entwinemedia.common</groupId>
+      <artifactId>functional</artifactId>
+    </dependency>
```

Comment:
What exactly do you need this for? I don't see any of its packages referenced anywhere. :thinking:

Reply:
For me everything compiles without this; which I would expect it to, as well. :thinking:

Reply:
Nevermind, I skipped the dependency plugin check. Seems to be necessary after all, if only to appease the check which might be incorrect in this case. :shrug: